src/changed-files.js

```diff
@@ -78,7 +78,4 @@ async function changedFiles({
   return changedFiles;
 }
 
-module.exports = changedFiles;
-Object.assign(module.exports, {
-  arePathsTheSame,
-});
+module.exports = changedFiles;
```

Comment:
I fixed a linting bug. Can you rebase again please?

Reply:
done